src/main/java/com/CezaryZal/entity/app/AuthenticationRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.CezaryZal.entity.app;
+
+import com.CezaryZal.entity.FormUser;
+
+public class AuthenticationRequest extends FormUser {
```

Comment:
Co daje dziedziczenie po FormUser?

Reply:
Jest to nadklasa, która posiada najważniejsze pola przy logowaniu, tworzeniu nowego użytkownika w tej aplikacji i HealthCalendar aplikacji.
W sumie to zamieniłem ją na klasę abstrakcyjną, będzie tak lepiej.